README.md

```diff
@@ -1,2 +1,16 @@
 # Archipelago.MultiClient.Net
 A client library for use with .NET based prog-langs for interfacing with Archipelago hosts.
+
+## Getting Started
+
+```csharp
+var hostUrl = "ws://localhost:38281";
+var session = ArchipelagoSessionFactory.CreateSession(hostUrl);
+
+session.Socket.Connect();
+
+var locationChecksPacket = new LocationChecksPacket();
+locationChecksPacket.Locations.Add(42);
+
+session.Socket.SendPacket(locationChecksPacket);
```

Comment:
I think these 3 lines should be session.Locations.CompleteLocationCheck(42);

Reply:
Yes, this was written before I added the location helper. Thanks for reminding me to update this.